tests/strategies/filter/test_crop_filter.py

```diff
@@ -40,10 +40,10 @@ def test_crop_filter(static_files: "Path") -> None:
 
     # Run pipeline = image_parser > crop_filter
     session = {}
-    session.update(crop_filter.initialize(session))
-    session.update(image_parser.initialize(session))
-    session.update(image_parser.get(session))
-    session.update(crop_filter.get(session))
+    session.update(crop_filter.initialize())
+    session.update(image_parser.initialize())
+    session.update(image_parser.get())
+    session.update(crop_filter.get())
```

Comment:
This whole sequence must be rethought.

The session is no longer given to the `initialize()` and `get()` methods. Rather, one might have to re-instantiate the original strategy with a configuration that utilizes the updates session...
So these lines will be slightly more involved...